src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -0,0 +1,58 @@
+package dk.cngroup.lentils.service;
+
+import org.springframework.stereotype.Service;
+
+import javax.imageio.ImageIO;
+import javax.swing.JFileChooser;
```

Comment:
Z tohohle jsem ted trochu zmateny. Ten dialog by mel byt pres prohlizec.
Tzn. v HTML ui toho input se nastavit, ze je to typ "file" a prohlizec ma na to pak vlastni widget a dialog.
Swing je u Desktop apek potreba nebo ne? @mjanousek

Reply:
No urcite mas pravdu, to jsem uplne neodhadla.   Nekdo to musi vyzkouset a rict: tudy ne, pratele...  :-)
Tak znovu a lip

Reply:
Tohle mas jakoby dialog, ale v te serverove aplikaci, ktery by se otevrel tam, ale uzivatel by ho nevidel, protoze sedi za internetem u jineho pocitace.

Takze se to udela v tom HTML, kde si to vyresi prohlizec a na server posle data toho souboru, ktery uzivatel vybere.